docs/topics/migrations.txt

```diff
@@ -784,6 +784,8 @@ Django can serialize the following:
 - Enumeration types (e.g. ``TextChoices`` or ``IntegerChoices``) instances.
 - Any Django field
 - Any function or method reference (e.g. ``datetime.datetime.today``) (must be in module's top-level scope)
+  - Functions may be decorated if wrapped properly, i.e. using :func:`functools.wraps`
+  - The :func:`functools.cache` and :func:`functools.lru_cache` decorators are also supported
```

Comment:
I'm not sure about this suggestion but I feel it would be a good idea to make it explicit that this support is not a "natural consequence" or implicit:
```suggestion
  - The :func:`functools.cache` and :func:`functools.lru_cache` decorators are explicitly supported
```

Reply:
Also this nested items seem to be rendering incorrectly:
![image](https://github.com/django/django/assets/124304/1748e92d-52e4-48ff-8ad5-2accf7238953)

Reply:
This should be fixed now. _(Forgot the blank lines around nested lists for Restructured Text 🤦🏻)_